teuthology/test/test_proc_regex.py

```diff
@@ -0,0 +1,56 @@
+import argparse
```

Comment:
Since we are testing `teuthology/orchestra/daemon/systemd` it is more convenient to add tests to `teuthogloy/orchestra/tests` or even `teuthology/orchestra/daemon/tests` the test name should be correspondingly `test_systemd.py`, the log excerpt can be located close to the test. For example, if to put tests to `teuthogloy/orchestra/tests`, it is reasonable to give test script like `test_daemon_systemd.py` and name the test case `test_pid()`.
The required files with dumps examples can be placed in subdirectory, for example `files/daemon-systemdstate-pid-ps-ef.output`

Reply:
updated.